adapters/libsdevent.h

```diff
@@ -0,0 +1,181 @@
+#ifndef __HIREDIS_LIBSDEVENT_H__
+#define __HIREDIS_LIBSDEVENT_H__
```

Comment:
```suggestion
#ifndef HIREDIS_LIBSDEVENT_H
#define HIREDIS_LIBSDEVENT_H
```

Leading underscores are technically an ISO violation.  We have other headers that violate the rule as well, which I am planning on getting to. :smile: